public/index.html

```diff
@@ -10,6 +10,14 @@
       content="Web site created using create-react-app"
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+
+    <!-- Component Library Stylesheet -->
+    <link rel="stylesheet" href="https://native-ui.netlify.app/native-ui.css">
+    <link rel="stylesheet" href="/src/index.css">
```

Comment:
linking CSS from `src` isn't appropriate.

Reply:
Ah yes. I forgot to change it as I took it form the last project. Thanks for pointing it out.